packages/zowe-explorer/src/trees/shared/SharedInit.ts

```diff
@@ -205,8 +205,8 @@ export class SharedInit {
                 )
             );
             context.subscriptions.push(
-                vscode.commands.registerCommand("zowe.saveSearch", (node: IZoweTreeNode) => {
-                    SharedTreeProviders.getProviderForNode(node).saveSearch(node);
+                vscode.commands.registerCommand("zowe.saveSearch", async (node: IZoweTreeNode) => {
+                    await SharedTreeProviders.getProviderForNode(node).saveSearch(node);
```

Comment:
Curious which style we prefer for an inline function that returns a `Promise<void>` 😋 

Would the following be equivalent?
```suggestion
                vscode.commands.registerCommand("zowe.saveSearch", (node: IZoweTreeNode) => {
                    return SharedTreeProviders.getProviderForNode(node).saveSearch(node);
```